resources/lang/en/patrols/general/hunting.json

```
@@ -3943,7 +4353,10 @@
             "cats_from": ["s_c"],
             "mutual": true,
             "values": ["romance", "like", "comfort"],
-            "amount": 10
+            "amount": 10,
+            "log": {
+                "cats_from": "from_cat got a bit too distracted by to_cat's scent while hunting, thankfully they both find it romantic."
```

```suggestion
                "cats_from": "from_cat got a bit too distracted by to_cat's scent while hunting. Thankfully, they both find it romantic."
```

being one sentence felt a little run-onish